work_dir_tools.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/sh
+
+#variables
+MODE=$1
+SDL_BACKUP=${TMPDIR-/tmp}"/sdL_backup"
+SDL=${2%/}
+
+#functions
+usage() {
+    echo "Usage: env.sh [OPTION] [PATH]"
+    echo "Path is a link to SDL bin folder"
+    echo "Mode can be the following:"
+    echo "  -b, --backup - Back up SDL configuration"
+    echo "  -c, --clean - Remove temporary SDL working files"
+    echo "  -r, --restore - Restore SDL configuration from backup"
+    echo "  -h, --help - Display this help"
+}
+
+backup() {
+	echo "Backup environment"
+	rm -r -f $SDL_BACKUP
+	mkdir $SDL_BACKUP
+	if [ -d $SDL_BACKUP ]; then
+		cp $SDL"/hmi_capabilities.json" $SDL_BACKUP
+		cp $SDL"/sdl_preloaded_pt.json" $SDL_BACKUP
+		cp $SDL"/smartDeviceLink.ini" $SDL_BACKUP
+		echo "Done"
+	else
+		echo "Folder '"$SDL_BACKUP"' doesn't exists"
+	fi
+}
+
+clean() {
+	echo "Clean environment"
+	if [ -d $SDL ]; then
+		rm -r -f $SDL"/storage"
+		rm -f $SDL"/app_info.dat"
+		rm -f $SDL"/backup_sdl_preloaded_pt.json"
+		rm -f $SDL"/ProtocolFordHandling.log"
+		rm -f $SDL"/SmartDeviceLinkCore.log"
+		rm -f $SDL"/TransportManager.log"
+		echo "Done"
+	else
+		echo "Folder '"$SDL"' doesn't exists"
+	fi
+}
+
+restore() {
+	echo "Restore environment"
+	if [ -d $SDL_BACKUP ]; then
+		cp $SDL_BACKUP"/hmi_capabilities.json" $SDL
```

Comment:
@LuxoftAKutsan : Additional verification on SDL folder existence will be implemented in a following commit